### DISCUSSION TITLE
Go big! Efficiently deploy and customize security tooling at enterprise scale [SEC1595B]

### DISCUSSION THREAD

**Discussion by ebickle** [community/community#141479] [Other Feature Feedback, Questions, & Ideas]
GitHub Universe 2024 Session: [Go big! Efficiently deploy and customize security tooling at enterprise scale](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1715360127550001lOUA)

> Whether you're a nimble team managing 30 repositories or an expansive enterprise with thousands, the rollout of new security tools can seem daunting. In this session, Eric Bickle, senior principal application security engineer at Asurion, will show you how easy it is to deploy and tune GitHub Advanced Security (GHAS) to your organization’s diverse needs. Eric will give you an in-depth look at how to deploy CodeQL at scale, respond to security alert dismissals, use Copilot Autofix to efficiently remediate existing code, and build a GitHub app for custom security workflows.

Looking forward to seeing you at [GitHub Universe 2024](https://githubuniverse.com/). If you have any questions, feedback, or ideas regarding the presentation, please feel free to add them in the comments below!

**Comment by Gasila5** [comment]
![IMG_20260201_011639_665](https://github.com/user-attachments/assets/0646b3a0-b5e8-4bc9-a68b-f1412d2b0dff)

**Reply by Gecko51** [reply]
Youhou ;)

**Comment by Gasila5** [comment]
0xaf4519c35952619E89C1ac2dE521c21Cbf891346

**Comment by Gasila5** [comment]
0xaf4519c35952619E89C1ac2dE521c21Cbf891346

**Comment by Gasila5** [comment]
0xaf4519c35952619E89C1ac2dE521c21Cbf891346

**Comment by Gasila5** [comment]
0xaf4519c35952619E89C1ac2dE521c21Cbf891346

**Comment by Gasila5** [comment]
0xaf4519c35952619E89C1ac2dE521c21Cbf891346

**Comment by Gasila5** [comment]
0xaf4519c35952619E89C1ac2dE521c21Cbf891346

**Comment by Gasila5** [comment]
0xaf4519c35952619E89C1ac2dE521c21Cbf891346

**Comment by Gasila5** [comment]
0xaf4519c35952619E89C1ac2dE521c21Cbf891346

**Comment by Gasila5** [comment]
0xaf4519c35952619E89C1ac2dE521c21Cbf891346

**Reply by Gecko51** [reply]
Is that an API key ?
